### PR TITLE
Dev/ncrocfer/bump flower

### PR DESCRIPTION
**Pull request by ncrocfer**
Just a small documentation to explain how to launch Flower.

I also updated the package version to fix a security issue: https://github.com/advisories/GHSA-q4qm-xhf9-4p8f (no breaking change in this version, tested locally).

### CODE REVIEW

**Review by ismailrei — APPROVED** [review]

File: docs/docs/guides/use-flower.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Use Flower
+
+Flower is a web based tool used to monitor and administrate your Celery clusters.
```
Comment (cerealito) [inline docs/docs/guides/use-flower.md:3]:
```suggestion
Flower is a web-based tool used to monitor and manage your Celery clusters.
```
explanation: https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=administrate

File: docs/docs/guides/use-flower.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+While Celery Director allows you to manage your entire workflows, Flower is able to give you more information at the tasks-level.
```
Comment (cerealito) [inline docs/docs/guides/use-flower.md:5]:
```suggestion
While Celery Director allows you to manage your entire workflows, Flower is able to give you more information at the task-level.
```
Explanation: https://www.englishclub.com/grammar/nouns-adjective.htm

File: docs/docs/guides/use-flower.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+It can also let you control your workers and their configuration (pool size, autoscale settings, terminate tasks...). Check the [documentation](https://flower.readthedocs.io/en/latest/) for the full list of features.
+
+!!!tip
+    So Flower is a great tool to used in parallel of Celery Director if you need more control of your tasks and your Celery clusters.
```
Comment (cerealito) [inline docs/docs/guides/use-flower.md:10]:
```suggestion
    So Flower is a great tool to use in parallel with Celery Director if you need more control of your tasks and your Celery clusters.
```

File: docs/docs/guides/use-flower.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Launch Flower
+
+The Flower tool is already included into Celery Director, so you just have to launch the following command to enable it:
+
+```
+$ director celery flower
+```
+
+You can now open the [http://localhost:5555](http://localhost:5555) url to track your tasks:
```
Comment (cerealito) [inline docs/docs/guides/use-flower.md:20]:
```suggestion
You can now open [http://localhost:5555](http://localhost:5555) to track your tasks:
```

**Review by cerealito — APPROVED** [review]

**Pull request merged**